website/docs/guides/PAK-upgrade-guide-1.10.0.html.markdown

```diff
@@ -19,7 +21,7 @@ In Terraform MongoDB Atlas Provider v1.10.0, some improvements were introduced w
 2. Use `project_assignment` in `mongodbatlas_project_api_key`: Instead of creating multiple mongodbatlas_project resources for each API Key assignment, you can now assign a PAK to multiple projects in a single resource block using the new `project_assignment` parameter in `mongodbatlas_project_api_key`.
 
 ## Examples
-We provide three examples in the atlas-api-key folder (under examples) to help you understand the new changes:
+We provide three examples in the [atlas-api-key folder](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/master/examples/atlas-api-key) (under examples) to help you understand the new changes:
```

Comment:
I cannot comment on the specific lines but I think the upgrade steps are problematic.   

Line 1 says to replace the api_key but with what?  Line 2?  If they just remove api_keys from project, which it says at the end of the line, that will unassign them from that project and cause some very upset customers.     You have to provide a guide for someone already using this method.  What is likely to be needed is:
1) remove the api_keys parameter from the state file.  After that is done remove it from the Terraform scripts.  This will preserve the existing assignments without actually removing them.  
2) Then they need to import in the key at the org level or at project and then use project to assign it (which will already be there, Terraform will now just reflect that reality).  Probably using mongodbatlas_project_api_key is the best bet.

@Zuhairahmed I would lean on Martin and Andrea for help here if you need it to ensure this guide gives a user a way to move from project to the new method with mongodbatlas_project_api_key without destroying their key assignments and potentially breaking a lot of scripts.  

Also, this still doesn't explain the nuance we've talked about - that org keys are assigned to projects and that even if you create a "project" key it's still an organization key.   It's not 100% needed but might be a nice FAQ to make this clear as it's not how it's represented in the UI and can be confusing.

Reply:
> Also, this still doesn't explain the nuance we've talked about - that org keys are assigned to projects and that even if you create a "project" key it's still an organization key. It's not 100% needed but might be a nice FAQ to make this clear as it's not how it's represented in the UI and can be confusing.

Added originally to only our [example folder README](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/atlas-api-key/README.md), but agree makes sense to add in here as well.

Reply:
@themantissa remaining updates have been flown through in case you want to take another look